install_config/registry/registry_known_issues.adoc

```diff
@@ -16,6 +16,18 @@ toc::[]
 The following are the known issues when deploying or using the integrated
 registry.
 
+[[known-issue-docker-registry-additional-load]]
+== Concurrent Build with Registry Pull-through
+
+The local *docker-registry* deployment takes on additional load. By default, it
+now caches content from *_registry.access.redhat.com_*. The images from
+*_registry.access.redhat.com_* for STI builds are now stored in the local
+registry. Attempts to pull them result in pulls from the local
+*docker-registry*. As a result, there are circumstances where extreme numbers of
+concurrent builds can result in timeouts for the pulls and the build can
+possibly fail. To alleviate the issue, scale the *docker-registry* deployment to
+more than one replica.
```

Comment:
Looks good.  The only addition might be a mention that the logs for the builder pod are where the timeouts can be seen.